tox.ini

```diff
@@ -40,4 +42,5 @@ deps =
     django22: django>=2.2,<2.3
     django30: django>=3.0,<3.1
     django31: django>=3.1,<3.2
-    djangomaster: https://github.com/django/django/archive/master.tar.gz
+    django32: django>=3.2b1,<3.3
```

Comment:
```suggestion
    django32: django>=3.2b1,<4.0
```

Reply:
Only if we'd change `django22: django>=2.2,<2.3` too. ;)

Reply:
yes we should